msbuild/Xamarin.Shared/Xamarin.Shared.targets

```diff
@@ -2464,13 +2283,195 @@ Copyright (C) 2018 Microsoft. All rights reserved.
 		</DetectDebugNetworkConfiguration>
 	</Target>
 
+
+	<!--
+
+		Post processing: this is what happens to the app bundle once it's built.
+
+		We must (the order here is important):
+
+		a) Run dsymutil on native binaries
+		b) Strip native binaries
+		c) Notify spotlight about our finished app bundle
+
+		A final step done later is to codesign the app bundle (and all its contained native binaries).
+
+		Generating debug symbols (dSYM directories) and stripping binaries.
+
+		* First we collect all the frameworks, dylibs and executable files
+		  from both the current project and any contained app extensions.
+		  Note: We don't iterate over all the *.framework and *.dylib files in
+		  the app bundle, we only look in the native references in the project
+		  file (one reason being that we need the metadata from those native
+		  references; although there are ways around that if need be).
+		* We write a list of all these items in a postprocessing.items file
+		  (so that container projects can load them if need be).
+		* If we're building the outermost app bundle (not an app extension nor
+		  a watch app, nor a .NET rid-specific app), we
+		  run dsymutil/symbolstrip in the executable project.
+			* If we're building a universal app in .NET, we don't run
+			  dsymutil/symbolstrip for each rid-specific build, but instead
+			  delay until we've created the universal app bundle.
+
+		* Metadata on items:
+			* NoSymbolStrip: set to true to not strip native symbols for this item (and false to dot it)
```

Comment:
%s/dot it/do it/gc

For some reason GH wouldn't let me add an inline patch here (maybe because commit by commit?)

Reply:
Yep, can only do patches in reviewing entire PR, not commit by commit.